### PR TITLE
ADD: See currency when tap and hold on balance

### PR DESCRIPTION
**Pull request by marcosrdz**
Screenshot 2023-11-12 at 8 53 54 PM

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/cllrpaqw5zxwl5bkkfy74dezgm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11873)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/sufvv2lubhxaddsh5e3madn6oi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11874)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: blue_modules/storage-context.js [hunk 1]
```diff
@@ -81,8 +81,9 @@ export const BlueStorageProvider = ({ children }) => {
   }, []);
 
   const getPreferredCurrency = async () => {
-    const item = await getPreferredCurrencyAsyncStorage();
+    const item = JSON.parse(await getPreferredCurrencyAsyncStorage());
```
Comment (Overtorment) [inline blue_modules/storage-context.js:84]: Weird. I don't remember putting json there.
Also, iirc string is parsed into string

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**